msal-auth-node/README.md

```diff
@@ -0,0 +1,129 @@
+Hello, world. You were able to access this because you provided a valid access token with the Greeting.Read scope as a claim.
+```
+
+<!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- Node.Js 16+
+
+## Setup
+
+### 1. Register your Web API
+
+Complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting    | Value for this sample app                                                    | Notes                                                                                              |
+|:---------------------------------:|:-----------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Name**                          | `Node Web API`                                                               | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**    | **Accounts in this organizational directory only (Single tenant)**   | Suggested value for this sample.                                                 |
+| **Platform type**              | _None_                                                               | No redirect URI required; don't select a platform.                               |
+| **Scopes defined by this API** | **Scope name**: `Greeting.Read`<br/>**Who can consent?**: **Admins and users**<br/>**Admin consent display name**: `Read API Greetings`<br/>**Admin consent description**: `Allows the user to see greetings from the API.`<br/>**User consent display name**: `Read API Greetings`<br/>**User consent description**: `Allows you to see greetings from the API.`<br/>**State**: **Enabled** | Required scope for this sample. |
+
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+
+
+
+### 4. Update code sample with app registration values
+
+```javascript
+app.use(jwt({
+  secret: jwks.expressJwtSecret({
+
+    // JWKS URI in the form of: https://login.microsoftonline.com/<tenant>/discovery/v2.0/keys
+    jwksUri: ''
+  }),
+  
+  // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+  audience: '',
+  
+  // Full directory URL, in the form of: https://login.microsoftonline.com/<tenant>/v2.0
+  issuer: '',
+
+  algorithms: ['RS256']
+}))
+```
+
+
+
+### 3. Install package(s)
+
+To install Node.Js libraries into your (virtual) environment:
```

Comment:
Make sure to port over any README suggestions from Marsh on your other PR over to this one.  Feel free to resolve this if you already have.

Reply:
@ckittel Readme.md has been updated.  I think I got all of Marsh's suggestions.